src/foam/nanos/session/Session.js

```diff
@@ -363,7 +363,8 @@ List entries are of the form: 172.0.0.0/24 - this would restrict logins to the 1
         // Support hierarchical SPID context
         var subX = rtn.cd(user.getSpid());
         if ( subX != null ) {
-          rtn = new OrX(reset(subX));
+          subX = reset(((foam.core.SubX) subX).fclone());
```

Comment:
@lchanmann can you elaborate here.  Wasn't expecting to see a clone of a context.

Reply:
subX were created and frozen after startup. When calling put on it, the subX will fclone itself then re-freeze again. adding fclone call explicitly here to prevent fclone/re-freeze on every put call.

Reply:
Can you add this as documentation here. I know I'm going to forget when I come back here in a few months. Thanks.

Reply:
Updated https://github.com/kgrgreer/foam3/pull/3043/commits/0c08795d94ea955669b990e3aca1f8459567ee04.